Umbraco-Cloud/Set-Up/Going-live/index.md

```diff
@@ -0,0 +1,26 @@
+---
+versionFrom: 7.0.0
+---
+
+# Going Live
+
+When you are about to go live with your website on Umbraco Cloud there is a few things you might want to consider beforehand.
+
+Below is a few article that will help you set up your project for going live.
+
+## [SMTP Settings](../SMTP-settings)
+In order for you to be able to send emails from your Cloud Project you will need to set up your SMTP settings. You might want to send emails through Umbraco Forms or invite Backoffice users.
+
+## [Configure/Add Hostnames](../Manage-Hostnames)
+When you are going live with your website you might want to have your visitors visit www.yourawesomewebsite.com instead of the Umbraco Cloud URL. 
+
+## [Deploy to Live](../../Deployment/Cloud-to-Cloud)
+The last step before your website is live and accessable to the public. Deploying from a Cloud environment to the Live environment.
+
+:::Tip
+If you would like to keep track of what goes on with your website after it has gone live you can set up a deployment webhook. You can read more about Webhooks in [this article](../../Deployment/Deployment-Webhook).
```

Comment:
Instead of using "this article", could you perhaps find some other phrasing for the link tekst? 🤔